fields/sql.go

```diff
@@ -119,9 +119,8 @@ type Scanner struct {
 	isValid bool
 }
 
-// CopyTo copies the scanner value to another reflect.Value. This is used for setting structs.
-func (s *Scanner) CopyTo(to reflect.Value) {
-	s.copy(s.value, to, s.isValid)
+func (s *Scanner) To(value reflect.Value) {
```

Comment:
Not sure if `To` name makes sense. Maybe `Reset` makes more sense?

One example I see often is having a sync.Pool of compression writers, like https://github.com/ungerik/go-pool/blob/master/gzip.go#L11:31. They almost always have a Reset method to replace underlying writer/destination.

Reply:
I'm open to renaming this. Maybe it should be `Target`? Also, now that I think about it, maybe I should clear the `value` at the end of `Scan`.